docs/rules/remove-redundant-type-cast.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Remove Redundant Type Casts
+ruleId: RemoveRedundantTypeCast
+since: 3.14.0
+minJavaVersion: 1.1
+remediationCost: 1
+links:
+description:
+    All method invocations of toString() on a String element are not needed. Applying this rule will remove such method calls.\n\nFor example, '"string".toString()' becomes '"string"'.\n\nThis transformation improves performance. RemoveToStringOnStringRule_name=Remove toString() on String.
```

Comment:
This description is not relevant for this rule. Better copy a (reduced) description of the rule from the section below.